AndroidSDKCore/src/main/java/com/leanplum/messagetemplates/BaseMessageDialog.java

```diff
@@ -129,17 +132,55 @@ protected BaseMessageDialog(Activity activity, boolean fullscreen, BaseMessageOp
           window.setDimAmount(0.7f);
         }
       } else {
+        if (htmlOptions == null) {
+          return;
+        }
+
         window.clearFlags(WindowManager.LayoutParams.FLAG_DIM_BEHIND);
-        window.setFlags(WindowManager.LayoutParams.FLAG_NOT_TOUCH_MODAL,
-            WindowManager.LayoutParams.FLAG_NOT_TOUCH_MODAL);
-        if (htmlOptions != null &&
-            MessageTemplates.Args.HTML_ALIGN_BOTTOM.equals(htmlOptions.getHtmlAlign())) {
-          dialogView.setGravity(Gravity.BOTTOM);
+        boolean treatAsBanner = treatAsBanner(htmlOptions);
+        if (treatAsBanner) {
+          // banners need to be positioned at the top manually (unless they get repositioned to the bottom later)
+          // or the anchor would be the center of the screen
```

Comment:
I know this is UI. But, can you please post, steps to reproduce this tests? So that we can add this to our release test book?

Reply:
Banners are already tested in QA, this change is for such a specific case, not sure if we need to QA it every time. But I'll add this to the QA playbook now.